qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/1.1.send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: 1.1.Send CV
+
+  Scenario: 1.1.01 positive sending CV
```

Comment:
In the name of each scenario, we need to use an underscore: for example 1.1_01, next one: 1.1_02
As in the screenshot
![image](https://user-images.githubusercontent.com/94701044/144028400-db194edc-7197-461a-a585-caf384bbe75f.png)

Reply:
done